### PR TITLE
Extract star wars skins

### PR DESCRIPTION
**Pull request by joshlory**
Begin moving Star-Wars-specific code to `/apps/src/studio/starwars`.

### CODE REVIEW

**Comment by joshlory** [comment]
PTAL!

File: apps/test/integration/levelSolutions/studio/ec_star_wars_grid.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+};
+
+export default {
+  app: 'studio',
+  skinId: 'hoc2015x',
+  levelDefinition: levelDef,
+  tests: [
+    {
+      description: 'Star Wars grid-aligned movement',
+      xml: `
```
Comment (joshlory) [inline apps/test/integration/levelSolutions/studio/ec_star_wars_grid.js]: Not 100% sure why I can't do this instead:

```js
editCode: true,
xml: 'moveRight(); moveRight();',
```

Comment (islemaster) [reply apps/test/integration/levelSolutions/studio/ec_star_wars_grid.js]: I'm not sure either.  What happens when you try?

Comment (joshlory) [reply apps/test/integration/levelSolutions/studio/ec_star_wars_grid.js]: BB-8 doesn't move from the starting position.  I tried following the pattern in the [ec_sandbox.js test](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/test/integration/levelSolutions/studio/ec_sandbox.js) but no luck.

Either way, good to have coverage of the `starwarsblocks` version as well.

Comment (Hamms) [reply apps/test/integration/levelSolutions/studio/ec_star_wars_grid.js]: `editCode` needs to be in the `levelDefinition`, not the test

**Comment by joshlory** [comment]
(Amended https://github.com/code-dot-org/code-dot-org/commit/c65bdea624404025370852ee8052b66d91b858b8)

**Pull request merged**